Modules/Poll/classes/class.ilObjPollGUI.php

```diff
@@ -326,26 +327,28 @@ public function render($a_form = null)
         $lng = $this->lng;
         $ilToolbar = $this->toolbar;
         $ilUser = $this->user;
-        $ui = $this->ui;
+        $ui_factory = $this->ui_factory;
+        $ui_renderer = $this->ui_renderer;
```

Comment:
Please remove these lines and refactor the usages to  $this->ui_factory and $this->ui_renderer